packages/ember-runtime/lib/ext/rsvp.js

```diff
@@ -61,7 +66,9 @@ export function onerrorDefault(e) {
         Test.adapter.exception(error);
         Logger.error(error.stack);
       } else {
-        throw error;
+        Ember.run.schedule(Ember.run.queues[Ember.run.queues.length-1], function() {
```

Comment:
I want to add a method of getting a promise for the next flush to the backburner refactor.